transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_cloud_servers.sql

```diff
@@ -0,0 +1,12 @@
+with mm_telemetry_prod_license as (
```

Comment:
No need for the CTE. The statement can work on its own.

Reply:
Removed CTE.